README.md

```diff
@@ -10,6 +10,7 @@ Contrastive learning of cell state dynamics in response to perturbations.
 > The CTC example configs are also tested on macOS with Apple M1 Pro SoCs (macOS 14.7).
 > Apple Silicon users need to make sure that they use
 > the `arm64` build of Python to use MPS acceleration.
+> Tested to work on Linux on the High Performance cluster, and may not work in other environments.
```

Comment:
This is the same as line 9. Also it won't be clear to the reader what 'the High Performance cluster' is.

Reply:
```suggestion
> Other system configurations may not work as intended.
```